packages/govuk-frontend/tasks/watch.mjs

```diff
@@ -57,5 +57,12 @@ export const watch = (options) => gulp.parallel(
     gulp.watch([
       `${slash(options.srcPath)}/govuk/**/*.{md,njk}`
     ], templates(options))
+  ),
+
+  // Copy GOV.UK Frontend static assets
+  task.name('copy:assets watch', () =>
+    gulp.watch([
+      `${slash(options.srcPath)}/govuk/assets/`
+    ], assets(options))
```

Comment:
I had noticed that when deleting a file from `src` it wasn't deleted from `package`, presumably because we're only running `copy:files` and not `clean`.

Removing the `unlink` wouldn't fix that but would prevent running the task when a file's deleted when it's not going to do anything anyway?